roles/image_builder/vars/main.yml

```diff
@@ -3,13 +3,13 @@
 microshift_image_blueprint_name: microshift
 microshift_image_blueprint_src_path: "/tmp/microshift.toml"
 microshift_image_compose_type: edge-commit
-microshift_image_pubkey_file: "~/.ssh/id_rsa.pub"
+microshift_image_pubkey_file: "{{ lookup('file', '~/.ssh/id_rsa.pub', errors='warn') }}"
```

Comment:
I think maybe we should change this to just `microshift_image_pubkey` since it's not really the file but instead is the string value of the pubkey which is what we get as the result of the lookup. That change would also need some docs touch ups as well.

Thoughts?